apps/web/components/Organization/Wrapper/Themes/TransportAuthorityTheme/TransportAuthorityHeader.tsx

```diff
@@ -45,6 +48,14 @@ const TransportAuthorityHeader: React.FC<HeaderProps> = ({
 
   const screenWidth = getScreenWidthString(width)
 
+  const { activeLocale } = useI18n()
+  const logoAltText = n(
```

Comment:
Just in case there isn't a title for the organization perhaps it'd be better to fallback to an empty string instead of potentially concatenating undefined or '' with ' Frontpage'